plugins/Kaleidoscope-Qukeys/src/kaleidoscope/plugin/Qukeys.h

```diff
@@ -55,6 +57,12 @@ constexpr Key LayerTapKey(uint8_t layer, Key tap_key) {
              (layer << 8) + tap_key.getKeyCode());
 }
 
+constexpr Key Qkey(Key primary_key, Key secondary_key) {
```

Comment:
Gonna need to find a way to fix the name collision with `constexpr Qukey(int8_t layer, KeyAddr k, Key alternate_key)` below, unless you're okay with keeping it like this.